src/schema/bookmarks.ts

```diff
@@ -517,6 +522,16 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
 
       return resolver(source, args, ctx, info);
     },
+    bookmarkList: async (
+      _,
+      { id }: { id: string },
+      ctx: AuthContext,
+    ): Promise<GQLBookmarkList> => {
+      return ctx.con.getRepository(BookmarkList).findOneByOrFail({
+        id,
+        userId: ctx.userId,
+      });
```

Comment:
What do we use this query for, can we use read replica?

Reply:
Just for the static props to get the relevant entity based on the query param. Definitely we can use read replica. Let me check some examples.

Reply:
Do you have some samples for this, can't seem to make it work with graphorm.

Reply:
![Screenshot 2024-12-10 at 7 57 03 PM](https://github.com/user-attachments/assets/99ff24df-e79e-4c9c-9403-f5119a17cba3)

Reply:
Does this mean I will just pass the `ctx` and not use the `queryRunner` completely?